appveyor.yml

```diff
@@ -23,17 +23,23 @@
         matrix:
             - BOND_BUILD: Python
               BOND_VS: "Visual Studio 14 2015"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 32
               BOND_BOOST: 59
             - BOND_BUILD: C++
               BOND_VS: "Visual Studio 14 2015 Win64"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 64
               BOND_BOOST: 59
             - BOND_BUILD: C#
               BOND_OUTPUT: Properties
               BOND_CONFIG: Debug
             - BOND_BUILD: Doc
             - BOND_BUILD: C++
               BOND_VS: "Visual Studio 12 2013"
-              BOND_BOOST: 55
+              BOND_VS_NUM: 12
+              BOND_ARCH: 32
```

Comment:
It's hitting the 1 hour limit; see https://ci.appveyor.com/project/sapek/bond/build/1.0.807